ui/src/utils/formatTaskMutation.js

```diff
@@ -0,0 +1,18 @@
+import upperSnakeCase from './upperSnakeCase';
+
+// This should be called just before sending a request to the
+// server when mutating a task payload.
+// The GraphQL server requires that enums are capitalized.
```

Comment:
This makes me uneasy -- we're transforming a task definition in the UI, hoping that we precisely reverse that transform in web-server before passing it on to the queue.  If our hopes aren't satisfied, then users will get strange errors about inputs they did not provide.  Let's keep an eye on this, and if things get more complicated perhaps consider alternative approaches that ferry task definitions intact between the queue and the UI?